router/customdestinationmanager/customdestinationmanager.go

```diff
@@ -174,7 +174,7 @@ func (customManager *CustomManagerT) onConfigChange(destination backendconfig.De
 		customManager.close(destination)
 	}
 
-	customDestination, err := customManager.newClient(destination.ID)
+	_, err := customManager.newClient(destination.ID)
```

Comment:
Let us remove the customdestination from the return of this function in its definition itself if its not used anywhere.

Reply:
The function only returns an error, customDestination is not being used.. Hence silenced it.

Reply:
ahh got it. Done